Makefile

```diff
@@ -26,8 +26,10 @@ __check_defined = \
     $(if $(value $1),, \
       $(error Undefined $1$(if $2, ($2))))
 
-all: tools build test fmtcheck vet validate_commits image ## Compiles fabric8-jenkins-proxy and runs format and style checks
+.PHONY: all
+all: tools build test fmtcheck vet lint validate_commits image ## Compiles fabric8-jenkins-proxy and runs format and style checks
 
+.PHONY: build
```

Comment:
I don't think this is correct. why do you think this should be phony?

Reply:
I am under an impression that 
Makefile targets are by default file targets. And if we are not using file targets we should use .PHONY
So here we don't have a file called build, thus .PHONY
Is that correct?